x-pack/plugin/autoscaling/src/test/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageDeciderServiceTests.java

```diff
@@ -355,39 +355,6 @@ public void testNodeLockSplitClone() {
         assertThat(createAllocationState(shardSizes, clusterState).maxNodeLockedSize(), equalTo(sourceSize * 2));
     }
 
-    public void testNodeSizeForDataBelowLowWatermark() {
```

Comment:
Hmm this test still has meaning I think, although it fails sometimes as currently implemented. Can we fix it rather than just removing it? At least we can assert that we're no more than 1 off?

Reply:
Hi @DaveCTurner , this test has been moved to org.elasticsearch.cluster.routing.allocation.DiskThresholdSettingsTests#testMinimumTotalSizeForBelowLowWatermark , that is why you see it removed here. Please unresolve this conversation if you need more on this.

Reply:
Ah sorry I missed that. Thanks for clarifying.